packages/web3torrent/src/components/share-list/ShareList.tsx

```diff
@@ -1,13 +1,24 @@
 import React from "react";
+import { RoutePath } from "../../routes";
 import { ShareFile } from "./share-file/ShareFile";
 import "./ShareList.scss";
 
-export type ShareListProps = { files: any[] };
+export type ShareListProps = { files: any[]; history: any };
```

Comment:
You can combine this type with `Pick<RouteComponentProps, "history">` to avoid using `any` here. Or, directly, `History` (from the `history` module).

You could also modify the `files` model to mimic the structure of a `File` upload, so when we switch to real files, the UI isn't affected. Or, at the very least, define the type for those mock files.

Reply:
Done!